lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -269,6 +269,10 @@ static void addMandatoryDiagnosticOptPipeline(SILPassPipelinePlan &P) {
     P.addDeadFunctionAndGlobalElimination();
   }
 
+  P.addConstGlobalVariableFoldingPass();
+  P.addOnoneSimplification();
```

Comment:
If guess you will not need this onone-simplification once you do the const-global folding in MandatoryPerformanceOptimizations